java/ql/src/utils/modelgenerator/internal/CaptureModelsSpecific.qll

```diff
@@ -27,9 +27,7 @@ private J::Method superImpl(J::Method m) {
 }
 
 private predicate isInTestFile(J::File file) {
-  file.getAbsolutePath().matches("%src/test/%") or
-  file.getAbsolutePath().matches("%/guava-tests/%") or
-  file.getAbsolutePath().matches("%/guava-testlib/%")
+  file.getAbsolutePath().matches(["%/test/%", "%/guava-tests/%", "%/guava-testlib/%"])
```

Comment:
It looks like this change affected the model generator test cases since those are stored in a `%/test/%` directory. (I wonder if that's part of the reason why the exclusion was hardcoded to `%src/test/%` before). 😄

```
1668 tests passed; 4 tests failed:
  FAILED: /home/runner/work/semmle-code/semmle-code/ql/java/ql/test/utils/modelgenerator/dataflow/CaptureSinkModels.qlref
  FAILED: /home/runner/work/semmle-code/semmle-code/ql/java/ql/test/utils/modelgenerator/dataflow/CaptureNeutralModels.qlref
  FAILED: /home/runner/work/semmle-code/semmle-code/ql/java/ql/test/utils/modelgenerator/dataflow/CaptureSourceModels.qlref
  FAILED: /home/runner/work/semmle-code/semmle-code/ql/java/ql/test/utils/modelgenerator/dataflow/CaptureSummaryModels.qlref
```

Would it make sense to update this to something like the below instead?

```suggestion
  file.getAbsolutePath().matches(["%/test/%", "%/guava-tests/%", "%/guava-testlib/%"]) and
  not file.getAbsolutePath().matches("%/ql/test/%") // allows our test cases to work
```